desktop/apps/consignments2/components/thank_you/index.styl

```diff
@@ -0,0 +1,39 @@
+.consignments2-submission-thank-you
+  max-width 1125px
+  text-align center
+  margin 0 auto
+
+  &__submitted-work
+    width 100%
+    background-color gray-lightest-color
+    height 120px
+    display flex
+    align-items center
+
+  &__submitted-message
+    garamond-size('l-body')
+    margin-right 20px
+
+  &__submitted-image img
+    max-height 68px
+    margin-left 40px
+    margin-right 40px
+
+  &__title
+    garamond-size('xl-headline')
+    margin-top 75px
+    margin-bottom 60px
+
+  &__next-steps
+    display flex
+    justify-content space-between
+    > *
```

Comment:
I would definitely the `> *` wildcard selector. It's a bit more explicit, but you can get the same effect by making each child tag of `div.next-steps` _also_  have a `b('next-step')` element class, in addition to their individual classes. That let's you style them both collectively and individually, while still keeping each selector a single class.